apluslms_roman/schemas/roman_project-v2.0.yaml

```diff
@@ -3,39 +3,45 @@ $schema: "http://json-schema.org/draft-04/schema"
 title: Course config
 type: object
 definitions:
+  image:
+    type: string
+    description: the docker image
   step:
     type: object
     additionalProperties: false
     required:
       - img
     properties:
-      img:
+      name:
         type: string
-        description: the docker image
+        description: name of the step
+        #doesn't allow names that are only numerals
+        pattern: "[^0-9]+"
+      img:
+        $ref: "#/definitions/image"
       cmd:
-        type: string
+        type: [string, array]
+        minItems: 1
+        items:
+          type: string
         description: commands to execute
       mnt:
         type: string
       env:
         type: object
-      name:
-        type: string
-        description: name of the step
-
-
 allOf:
   - $ref: version-v1.0
-
 additionalProperties: false
 properties:
   version: {}
   steps:
     type: array
+    minItems: 1
```

Comment:
Based on commit https://github.com/apluslms/roman/pull/9/files#r294316902, we need to revisit this idea.